lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -316,8 +316,13 @@ void StagesSubject::onStageInvalidate(const MayaUsdProxyStageInvalidateNotice& n
     afterOpen();
 
 #ifdef UFE_V2_FEATURES_AVAILABLE
-    Ufe::SceneItem::Ptr sceneItem = Ufe::Hierarchy::createItem(notice.GetProxyShape().ufePath());
-    Ufe::Scene::instance().notify(Ufe::SubtreeInvalidate(sceneItem));
+    auto p = notice.GetProxyShape().ufePath();
```

Comment:
It is possible for this path to return empty when called before the proxy shape is ready. The MDagPath in the proxy shape is empty. The previous code (before my world cleanup) would thus return a Ufe::path to "|world" which was incorrect.